saritasa_invocations/django.py

```diff
@@ -73,6 +73,23 @@ def check_new_migrations(context: invoke.Context) -> None:
     )
 
 
+@invoke.task
+def startapp(context: invoke.Context):
+    """Create new django app."""
```

Comment:
Let's add a note about cookiecutter to docstring (so it may be viewed by `inv django.startapp -h`

Reply:
In other use write it like requires: something and link to link, @Eg0ra you could check it in other places